src/Teams/Data/Repositories/ProgramCodeQuestionRepository.cs

```diff
@@ -18,5 +20,10 @@ public ProgramCodeQuestion PickById(Guid id)
         {
             return _db.ProgramCodeQuestions.Single(q => q.Id == id);
         }
+        public void Add(ProgramCodeQuestion question)
+        {
+            _db.ProgramCodeQuestions.Add(question);
+            _db.SaveChanges();
```

Comment:
Could you please remove saving from repository? With current implementation every call of Add will also call SaveChanges. There is no way to call Add twice and save changes in one database transaction. This is not a desired behavior.